README.md

```diff
@@ -127,32 +129,32 @@ A0SimpleKeychain().deleteEntry(forKey: "auth0-user-jwt")
 
 ## Contributing
 
-Just clone the repo, and run `pod install` from the Example directory and you're ready to contribute!
+Just clone the repo, run `carthage bootstrap` and you're ready to contribute!
 
-## Issue Reporting
+## What is Auth0?
 
-If you have found a bug or if you have a feature request, please report them at this repository issues section. Please do not report security vulnerabilities on the public GitHub issue tracker. The [Responsible Disclosure Program](https://auth0.com/whitehat) details the procedure for disclosing security issues.
+Auth0 helps you to:
 
-## License
+* Add authentication with [multiple authentication sources](https://auth0.com/docs/identityproviders), either social like **Google, Facebook, Microsoft Account, LinkedIn, GitHub, Twitter, Box, Salesforce, amont others**, or enterprise identity systems like **Windows Azure AD, Google Apps, Active Directory, ADFS or any SAML Identity Provider**.
```

Comment:
Seems redundant to mention 'authentication' twice here. Also a couple of edits and typos:

```suggestion
* Add authentication with [multiple sources](https://auth0.com/docs/identityproviders), either social identity providers such as **Google, Facebook, Microsoft Account, LinkedIn, GitHub, Twitter, Box, Salesforce** (amongst others), or enterprise identity systems like **Windows Azure AD, Google Apps, Active Directory, ADFS, or any SAML Identity Provider**.
```